core/vibe/core/core.d

```diff
@@ -691,8 +691,8 @@ public void setupWorkerThreads(uint num = logicalProcessorCount())
 public @property uint logicalProcessorCount()
 {
 	version (linux) {
-		import core.sys.posix.unistd;
-		return cast(uint) sysconf(_SC_NPROCESSORS_ONLN);
+		import std.parallelism : totalCPUs;
+		return totalCPUs;
 	} else version (OSX) {
```

Comment:
Why don't you use it for the other OSes as well?

Reply:
Sure, why not! Thanks pointed out!